src/db.c

```diff
@@ -325,7 +325,7 @@ create_thing(dbref player, const char *name, dbref location)
     PUSH(newthing, CONTENTS(location));
 
     EXITS(newthing) = NOTHING;
-    SETVALUE(newthing, 1);
+    SETVALUE(newthing, 0);
```

Comment:
TL;DR The stock @ CREATE still behaves the same after this change with the exception of an edge case that would cause a -1 value thing object.

That highlighted change to db.c create_thing() doesn't affect the @ create command.  The @ CREATE command function (create.c do_create) does call create_thing() but it immediately changes the value with SETVALUE.

I left that unchanged, with the exception of adding in a check that makes sure it never creates a negative value object. (if @ tune object_cost = 0 then due to the way the endowment calculation works, it actually tries to give the object a value of -1.) I don't think there are any MUCKs that will rely on that behavior.

I also hunted through the code looking for calls to create_thing to see if anything relies on that setting of the value to 1, and the only case was the NEWOBJECT primative.

Reply:
Still with these changes:
```
> @tune object_cost = 10
Parameter set.
(int)  object_cost          = 10
> @create testx
Object testx(#189) created.
> examine testx
testx(#189)  Owner: Mercury  Value: 1
Type: THING
Created:  Sun Sep 15 22:13:20 2019 EDT
Modified: Sun Sep 15 22:13:20 2019 EDT
Lastused: Sun Sep 15 22:13:20 2019 EDT
Usecount: 0
[ Use 'examine <object>=/' to list root properties. ]
Memory used: 228 bytes
Home: Test(#183R)
Location: Mercury(#164PWBC)
No actions attached.
```

Reply:
Awesome work, well done! :)

Reply:
Unfortunately, yes, newexit, newroom, setlink, and setlink_array do still have the some problem.

But unlike thing objects, they don't store their penny value on an object-by-object basis in the database :( So there really is no straightforward way to prevent `@recycle` from giving you free pennies for them, in the same way as for thing objects.